SofascoreAcademy/SofascoreAcademy/Controllers/EventsViewController.swift

```diff
@@ -55,21 +52,24 @@ extension EventsViewController: UITableViewDataSource, UITableViewDelegate {
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        events[tournaments[section].countryName]?.count ?? 0
+        events[tabIndex][tournaments[section].countryName]?.count ?? 0
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCell(withIdentifier: "EventTableCell", for: indexPath) as? EventTableCell
         else {
             return UITableViewCell()
         }
-        let event = events[tournaments[indexPath.section].countryName]?[indexPath.row] ?? EventModel.emptyEventModel
+        guard let event = events[tabIndex][tournaments[indexPath.section].countryName]?[indexPath.row] else { return UITableViewCell() }
         
         cell.set(eventModel: event)
-
         return cell
     }
     
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        self.eventViewTapped()
```

Comment:
Ovdje još jednio bi trebalo ili možeš to i kasnije u zadaćama... vidjeti na temelju indexa o kojem se eventu radi pa samo u eventDetails proslijediti eventID